scripts/upgrade_full_clean.sh

```diff
@@ -25,9 +25,12 @@ docker volume prune -f
 # Remove the workspace volume
 docker volume rm prover-node-docker_workspace-volume
 
-# Remove the mongodb volume
+# Remove the mongodb volume (legacy)
```

Comment:
Can add scripts "docker volume rm prover-node-docker_mongodb_data &> /dev/null" in the beginning of upgrade.sh to help prover remove mongo db to release disk